app/views/retros/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<p id="notice"><%= notice %></p>
```

Comment:
Are these files needed at this point? `edit.html.erb``index.html.erb``_retro.json.jbuilder`